internal/service/alarms/internal/db/models/converters.go

```diff
@@ -137,3 +137,12 @@ func ConvertAlertSubToNotificationSub(as *AlarmSubscription) *notifier.Subscript
 	}
 	return &info
 }
+
+// ConvertDefToPC probableCause is subset of alarm definition
+func ConvertDefToPC(def AlarmDefinition) api.ProbableCause {
```

Comment:
Could we just spell this out to `ConvertDefinitionToProbableCause`.  It won't cost more $$.  ;)

Reply:
true It is free!! Ok done :) (also update other spots where it says "def")